Assets/Scripts/StageManager.cs

```diff
@@ -82,27 +86,30 @@ protected virtual void OnSceneLoaded(Scene scene, LoadSceneMode mode)
         Time.timeScale = 1;
         isGameOver = false;
     }
-    public virtual GameObject ActiveCharacter() 
+    public virtual GameObject ActiveCharacter()
     {
         return activeCharacter;
     }
-    protected virtual void PauseGame()
+    public virtual void PauseGame()
     {
-        pauseMenu.SetActive(true);
+        if (pauseMenu != null)
+            pauseMenu.SetActive(true);
```

Comment:
```suggestion
        pauseMenu?.SetActive(true);
```
똑같이 추가하신 부분들도 다 이렇게 적용해주면 좋을 것 같네요